partner_contact_birthplace/views/res_partner.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="view_personal_information_birthplace" model="ir.ui.view">
+        <field name="name">Birthplace field</field>
+        <field name="model">res.partner</field>
+        <field
+            name="inherit_id"
+            ref="partner_contact_personal_information_page.personal_information"
+        />
+        <field name="arch" type="xml">
+            <xpath expr="//group[@name='personal_information_group']">
+                <field name="birth_city" />
```

Comment:
I'm wondering why this order?

Shouldn't it country first, then state and city ?

![image](https://github.com/user-attachments/assets/65e02857-9411-4733-8316-be11544c4d6b)

Reply:
@SirAionTech

Reply:
I probably copied the same order that is used in the partner address https://github.com/odoo/odoo/blob/29405bb41516a57b635c1b8cedbfa564d626b964/odoo/addons/base/views/res_partner_views.xml#L138-L141, but it can be changed indeed if you prefer another order.